pom.xml

```diff
@@ -152,6 +152,25 @@
               </dependency>
             </dependencies>
           </plugin>
+          <!-- Generate test DB schema without key `UQ_MUTATION_EVENT_DETAILS` to accommodate H2 MySQL driver not supporting partial indexes -->
+          <plugin>
+            <artifactId>exec-maven-plugin</artifactId>
+            <groupId>org.codehaus.mojo</groupId>
+            <version>1.6.0</version>
+            <inherited>false</inherited>
+            <executions>
+              <execution>
+                <id>Generate Test DB schema</id>
+                <phase>integration-test</phase>
```

Comment:
Will this work for both integration-test testing and regular unit testing? .. I'm not familiar with the integration-test phase and thought that our test database is also in use during plain unit tests.